### PR TITLE
Poll /transaction instead of waiting for callbacks

### LINKED ISSUE #71: Remove postMessage callback, support /transaction polling
We should poll the anchor's `/transaction(s)` endpoint instead of waiting for them to send a callback

related to stellar/integration-meta#258

### PR DESCRIPTION
**Pull request by JakeUrban**
resolves #71 

I also replaced the use of `axios` in `withdrawAsset.ts` to be consistent with the rest of the project.

### CODE REVIEW

File: src/components/wallet/methods/depositAsset.ts [hunk 1]
```diff
@@ -157,40 +153,89 @@ export default async function depositAsset(
...
+            break
+          }
+        }
       }
+      // run loop every 2 seconds
+      await new Promise((resolve) => setTimeout(resolve, 2000))
     }
+
+    this.updateAccount()
+    this.logger.instruction('Transaction status complete')
```
Comment (Zagan202) [inline src/components/wallet/methods/depositAsset.ts:237]: Before communicating that the transaction is complete I believe we should check if the `currentStatus` is `complete` and if `popup.closed`.
If I were to close the pop up before its complete the logger states that it is.

**Review by Zagan202 — CHANGES_REQUESTED** [review]
Looks good and pretty much behaves as expected.
Only need to resolve the merge conflict and address the comment I made.
Good work!

File: src/components/wallet/methods/depositAsset.ts (hunk 1 above)
Comment (JakeUrban) [reply src/components/wallet/methods/depositAsset.ts]: Done!

File: src/components/wallet/methods/withdrawAsset.ts [hunk 2]
```diff
@@ -113,82 +88,123 @@ export default async function withdrawAsset(
...
+              'There was a problem processing your transaction'
+            )
+            break
+          }
+        }
       }
+      // run loop every 2 seconds
+      await new Promise((resolve) => setTimeout(resolve, 2000))
     }
+
```
Comment (Zagan202) [inline src/components/wallet/methods/withdrawAsset.ts:205]: We should include the same  logger instruction here in withdraw asset
```suggestion
    this.logger.instruction(`Transaction status: ${currentStatus}`)
    if (!['completed', 'error'].includes(currentStatus) && popup.closed) {
      this.logger.instruction(
        'The popup was closed before the transaction reached a terminal status, ' +
          'if your balance is not updated soon, the transaction may have failed.'
      )
    }
```

**Review by Zagan202 — CHANGES_REQUESTED** [review]

**Review by Zagan202 — APPROVED** [review]

**Pull request merged**